index.html

```diff
@@ -9,51 +9,235 @@
+            <p class="main-paragraph">Hello, I'm a software developer! I can help you build a product, feature or website. 
+                Look through some of my work and experience! If you like what you see and have a project you need coded,
+                 don't hesitate to contact me.
+            </p>
+            <p class="connect">LET'S CONNECT</p>
+        </div>
+        
+        <div class="headline-section">
+            <ul>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/twiter.png" alt="Twitter"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social linkendIn">
+                        <img src="./images/Linkedinicon.png" alt="LinkendIn"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/mail.png" alt="Email"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social github">
+                        <img src="./images/Vector.png" alt="GitHub"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social peace">
+                        <img src="./images/fingers.png" alt="Peace"/>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <button type="button" class="get-resume">Get my resume</button>
+
+        <!--Languages-Skills Section-->
+        <div class="skills-lang">
+            <ul>
+                <li id="languages">
+                    <a href="#" class="languages">
+                        <span class="lang-head">Languages</span> <img src="./images/Chevron-Down.png" alt="Chevron-Down"/>
+                    </a> 
+                </li>
+                <li class="about-lang">
+                    <img src="./images/javaScript-icon.png" alt="JavaScript"/>
+                    <a href="#" class="lang javascript">JavaScript</a> 
+                </li>
+                <li class="about-lang">
+                    <img src="./images/html-icon.png" alt="Html"/>
+                    <a href="#" class="lang html">HTML</a> 
+                </li>
+                <li class="about-lang">
+                    <img src="./images/css-icon.png" alt="CSS"/>
+                    <a href="#" class="lang css">CSS</a> 
+                </li>
+                <li class="framework-skills">
+                    <a href="#" class="framework">Framework</a>
+                    <img src="./images/Chevron-Right.png" alt="Right-caret"/>
+                </li>
+                <li class="framework-skills">
+                    <a href="#" class="skills">Skills</a>
+                    <img src="./images/Chevron-Right.png" alt="Right-caret"/>
+                </li>
+            </ul>
+        </div>
     </section>
```

Comment:
- I would advise you to follow the `about-me` from the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) section as required, buttons, fonts, size and colours should be similar. Also, add the `border-radius` on the top-left of that section as suggested by the previous reviewer.
- Please, follow the design  below
![image](https://user-images.githubusercontent.com/80612925/170782513-f61836b4-9a40-47b1-a1b6-c8070e59d3c3.png)